src/FsAutoComplete/CommandResponse.fs

```diff
@@ -43,6 +43,47 @@ module internal CompletionUtils =
     | FSharpEnclosingEntityKind.Enum -> "En"
     | FSharpEnclosingEntityKind.DU -> "D"
 
+module internal ClassificationUtils =
+
+  // See https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#semantic-token-scope-map for the built-in scopes
+  let map (t: SemanticClassificationType) : string =
```

Comment:
@Krzysztof-Cieslak can you spot-check the decisions I made here for the new DU cases? I tried to stick to the well-known tags called out in the API link above.